apps/prairielearn/src/lib/question-testing.ts

```diff
@@ -391,14 +425,15 @@ export async function startTestQuestion(
   question: Question,
   course_instance: CourseInstance | null,
   course: Course,
+  user_id: string,
   authn_user_id: string,
 ): Promise<string> {
   let success = true;
   const test_types: TestType[] = ['correct', 'incorrect', 'invalid'] as const;
 
   const serverJob = await createServerJob({
     courseId: course.id,
-    userId: String(authn_user_id),
+    userId: String(user_id),
     authnUserId: String(authn_user_id),
```

Comment:
AFAICT the use of the `String(...)` constructor here is unnecessary, since the types declare that this is already a string.

Reply:
Removed `String(...)` in [bc258cb](https://github.com/PrairieLearn/PrairieLearn/pull/10840/commits/bc258cb63105129c9382b9367281d7b040a14698).

Reply:
Thanks! There's one on the following line as well, would be nice to clean that up while we're here.

Reply:
Removed in [61115af](https://github.com/PrairieLearn/PrairieLearn/pull/10840/commits/61115af1d991178079dbce0835d46b295561be0b).